pkg/fakedata/generator.go

```diff
@@ -177,6 +178,27 @@ var enum = func(column Column) string {
 	return withEnum(enum)(column)
 }
 
+var fileCache map[string][]string
+
+var file = func(column Column) string {
```

Comment:
It's funny. I think it's literally "historical reasons" even though the project is so young. I started it off an existing codebase and evolved the design in a direction I liked more. So long story short, there's no good reason right now. I'll put in a small gardening/good first patch issue for it.

I think it's good we keep it consistent right now though so I appreciate you followed existing style.